app/script/entity/Conversation.js

```diff
@@ -394,19 +394,15 @@ z.entity.Conversation = class Conversation {
   getNumberOfClients() {
     const participantsMapped = this.participating_user_ids().length === this.participating_user_ets().length;
     if (participantsMapped) {
-      let usersWithoutKnownClients = 0;
-      const numberOfKnownClients = this.participating_user_ets().reduce(userEt => {
-        if (!userEt.devices().length) {
-          usersWithoutKnownClients = usersWithoutKnownClients + 1;
+      return this.participating_user_ets().reduce(userEt => {
+        if (userEt.devices().length) {
+          return userEt.devices().length;
         }
-        return userEt.devices().length;
+        return z.client.ClientRepository.CONFIG.AVERAGE_NUMBER_OF_CLIENTS;
       }, this.self.devices().length);
-
-      const estimatedUnknownClients = usersWithoutKnownClients * 4;
-      return numberOfKnownClients + estimatedUnknownClients;
     }
 
-    return this.get_number_of_participants() * 4;
+    return this.get_number_of_participants() * z.client.ClientRepository.CONFIG.AVERAGE_NUMBER_OF_CLIENTS;
```

Comment:
life is so much nicer without the magic numbers.. I doubt if anyone would know what is this 4 in a year from now..

Reply:
if not in a month :D